src/Config/Validator/Deploy/PostDeploy.php

```diff
@@ -16,6 +16,8 @@
  */
 class PostDeploy implements ValidatorInterface
 {
+    private const URL = 'https://devdocs.magento.com/cloud/project/project-conf-files_magento-app.html#hooks';
```

Comment:
I am not sure that we need to add links to devdocs as this may be changed from time to time

Reply:
This is probably the best we have since hooks will also change over time

Reply:
@meker12 any thoughts?

Reply:
@shiftedreality , why don't we just take link to magento-cloud template then? It is always up to date.

Reply:
It's currently not up-to-date

Reply:
@shiftedreality what do you mean that it is not up to date?

Reply:
It does not have scenarios right now, there is a task to update https://jira.corp.magento.com/browse/MCLOUD-5717

Generally, I think it's not best place as a source of truth since it's not one-line change to make.